src/Parser/Shaman/Restoration/Modules/Features/TidalWaves.js

```diff
@@ -46,6 +46,34 @@ class TidalWaves extends Analyzer {
       });
   }
 
+  get suggestionThresholds() {
```

Comment:
I recommend also using these threshold values in the `suggestions()` method so that you have 1 place you need to update should you ever want to change these. Also try to use the `actual` property there if possible.